R/function_left_parentheses_linter.R

```diff
@@ -41,11 +41,11 @@
 #' - [spaces_left_parentheses_linter()]
 #' @export
 function_left_parentheses_linter <- function() { # nolint: object_length.
-  xpath <- "
-  //FUNCTION[@col2 != following-sibling::OP-LEFT-PAREN/@col1 - 1]
-  |
-  //SYMBOL_FUNCTION_CALL/parent::expr[@col2 != following-sibling::OP-LEFT-PAREN/@col1 - 1]
-  "
+  pos_xpath <- xp_or(
```

Comment:
The variable name seems odd to me. Maybe add `bad` somewhere?

Reply:
Should be clearer in the new version.